app/assets/v2/js/grants/index.js

```diff
@@ -20,10 +20,10 @@ $(document).ready(() => {
     $(e.currentTarget).toggleClass('turn');
   });
 
-  waitforWeb3(function() {
-    const linkURL = etherscan_tx_url($('#tx_id').val());
-    
-    $('#tx_link').attr('href', linkURL);
+  waitforWeb3(() => {
+    $('#sub_tx_link').attr('href', etherscan_tx_url($('#sub_tx_link').attr('href')));
```

Comment:
seems to me that the etherscan URLs are dependant upon the web3 network that the grant is on, not the ones that the users browser is on

Reply:
@owocki addressed it ! It takes it all based on the network in which the grant was created